src/main/resources/db/migration/V1_0__add_database_schema.sql

```diff
@@ -1,5 +1,6 @@
 
     create table document (
+        archive bit,
```

Comment:
not null?

Reply:
Sure why not. The attribute is defined as primitive boolean in entity, but I guess it's good to have extra protection when/if people do direct insertions in database. 👍

Reply:
Yes, and to protect the database, in case someone refactors the code and introduce a "Boolean" in the code.